server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -112,9 +112,10 @@
     public static final Version V_4_4_3 = new Version(7_04_03_99, false, org.apache.lucene.util.Version.LUCENE_8_7_0);
 
     public static final Version V_4_5_0 = new Version(7_05_00_99, false, org.apache.lucene.util.Version.LUCENE_8_7_0);
-    public static final Version V_4_5_1 = new Version(7_05_01_99, true, org.apache.lucene.util.Version.LUCENE_8_7_0);
```

Comment:
This is just a humble note. Was it intended to remove this line?

Reply:
yes. It will be re-added once we release 4.5.1 and sync the release notes / versions.

Could also have left it in, it doesn't really matter.